packages/isomorphic/src/internal/normInput.ts

```diff
@@ -0,0 +1,11 @@
+import { Input } from './index'
```

Comment:
Importing from `./index` feels weird to me - perhaps the types can be moved to a separate file (`types.ts`).

Reply:
fair